src/model/BackgroundImage.h

```diff
@@ -11,44 +11,47 @@
 
 #pragma once
 
-#include <XournalType.h>
+#include "XournalType.h"
 
 #include <gtk/gtk.h>
 
-class BackgroundImageContents;
+#include <memory>
 
-class BackgroundImage
+struct BackgroundImageContents;
```

Comment:
Since this is internal, put it in a private section in `BackgroundImage`.

Reply:
You are right, already applied it :).